scaffolds/cpp/README.md

```diff
@@ -0,0 +1,55 @@
+# C++ Scaffold for Code Retreat
+
+This code scaffold uses [CxxTest](http://cxxtest.com). We recommend using the
+docker image [euranova/cxxtest](https://github.com/euranova/docker-cxxtest), but
+installing `cxxtest` with the package manager is also an option.
+
+## Start docker image
+
+Start a bash prompt in the docker image, with the code mounted as a volume:
+
+```bash
+docker run --user ${UID} -it -v ${PWD}:/app euranova/cxxtest
```

Comment:
I would create a wrapper to make it transparent.

Reply:
Done. It's called `docker.sh`, just like for the C# scaffold